File: JavaProjects/JavaLanguageExamples/input/sampleScript.rb
class Employee
  attr_reader :salary
  def initialize(name,age,salary)
    @name = name
    @age = age
    @salary = salary
    @addresses =[]
  end
  def awardBonus(bonus)
    @salary += bonus
  end
  def addAddress(address)
    @addresses.push(address)
  end
  def locations()
    msg = "Lives at "
    @addresses.each {|a| msg << a << " " }
    return msg
  end
  def to_s
    return "#@name of age #@age earning #@salary"
  end
end

e = Employee.new("Dave Smith",32,35000)
puts e.inspect()
puts e.salary
puts e

e.addAddress("10 Arcatia Road")
e.addAddress("11 Privet Drive")
puts e.locations()

puts e.inspect()

File: JavaProjects/JavaLanguageExamples/input/sampleScript.js
	function printEmployee() {
		var msg = "Empoyee called ";
		msg += this.name;
		msg += " working in ";
		msg += this.dept;
		msg += " earning ";
		msg += this.salary;
		msg += "\n";
		return msg;
	}
	function Employee(p1,p2,p3) {
		this.name = p1;
		this.dept = p2;
		this.salary = p3;
		this.toString = printEmployee
		this.awardBonus = function(bonus) {this.salary += bonus;}
	}

	var e1 = new Employee("Homer","IT",30000);
	var e2 = new Employee("Marge","HR",40000);

	print("Employees Before Bonus\n");
	print(e1);
	print(e2);

	e1.awardBonus(500);
	e2.awardBonus(1000);

	print("Employees After Bonus\n");
	print(e1);
	print(e2);